### PR TITLE
extracted from #1617

### PR DESCRIPTION
**Pull request by hhugo**
#1617

### CODE REVIEW

**Comment by hhugo** [comment]
cc @OlivierNicole

File: compiler/lib/source_map.ml [hunk 1]
```diff
@@ -19,6 +19,16 @@
 
 open! Stdlib
 
+module Source_content = struct
```
Comment (OlivierNicole) [inline compiler/lib/source_map.ml:22]: Nit: naming it `Sources_content` (with an S to Sources) would match the sourcemap field name `sourcesContent`.

**Review by OlivierNicole — APPROVED** [review]
LGTM, thanks for extracting this.

File: compiler/lib/source_map.ml (hunk 1 above)
Comment (hhugo) [reply compiler/lib/source_map.ml:22]: The sourcemap field store multiple sources but `Source_content.t` is about a single one.

Comment (OlivierNicole) [reply compiler/lib/source_map.ml:22]: Ah, true. Never mind me then.

**Pull request merged**